### ISSUE TITLE
Extract Femah.Core.Providers.SqlServerProvider in to a separate repo

### ISSUE THREAD

**Issue by lholman** [lloydstone/femah#102]
Extract the `Femah.Core.Providers.SqlServerProvider` out of https://github.com/lloydstone/femah in to a separate repo (with any supporting scripts and documentation).

Should we do this prior to MVP or post, I figure it will be a bit nasty to rip it away from people and force them to upgrade if they've started using it.

**Comment by hotstone** [comment]
@lholman  Yep, agreed.

**Comment by lholman** [comment]
I agree too, think we need to, will be a nasty experience for any early adopters if we then decide to remove it out of Femah.Core and force them to then add another Nuget package reference and all that.

**Comment by lholman** [comment]
@hotstone Call it 

Femah.Providers.Sql 
or
Femah.Sql

Or SqlServer instead of Sql?

**Comment by hotstone** [comment]
@lholman Femah.SqlServer I say!

**Comment by AwaNoodle** [comment]
+1 for Femah.SqlServer

Anything I get on with for this?

**Comment by lholman** [comment]
@AwaNoodle I'd like to get #99 underway, interested? This one really requires another repo in lloydstone being created and a simple lift and plonk in to that (probably easiest for me to do that) and  then some build shenanigans will be needed (duplicate of what we do now), interested in the build bit?

**Comment by AwaNoodle** [comment]
@lholman sure thing. I guess this isn't dependent on the move to a separate package.

**Comment by lholman** [comment]
Relies on either accepting or declining #105 and #107 
/cc @hotstone

**Comment by hotstone** [comment]
Will do these later today!
